firmware/steering/src/init.cpp

```diff
@@ -49,4 +50,13 @@ void init_communication_interfaces( void )
 
     DEBUG_PRINT( "init Control CAN - " );
     init_can( g_control_can );
+
+    // Filter CAN messages - accept if (CAN_ID & mask) == (filter & mask)
+    // Set buffer 0 to filter only brake module and global messages
```

Comment:
Trivial comment: this is the steering module but the comment mentions that this is the brake module.

Reply:
Yep, comment needs updated.